src/components/DocumentationTopic.vue

```diff
@@ -259,6 +267,11 @@ export default {
       type: String,
       required: true,
     },
+    viewMoreLinkTitle: {
```

Comment:
Since it's a pretty simple component, I'm wondering if a better alternative to making the title configurable might be to just use a themed import for the `ViewMore` component—that way, the whole component could be easily overwritten (and it's a pretty simple component with mostly just this text). That would also avoid the mess of having to pass this prop down from the root component.

What do you think?